src/libcrun/container.c

```diff
@@ -975,11 +954,11 @@ maybe_chown_std_streams (uid_t container_uid, gid_t container_gid,
   return 0;
 }
 
-static inline int
-notify_handler (struct container_entrypoint_s *args,
-                enum handler_configure_phase phase,
-                libcrun_container_t *container, const char *rootfs,
-                libcrun_error_t *err)
+int
+libcrun_container_notify_handler (struct container_entrypoint_s *args,
+                                  enum handler_configure_phase phase,
+                                  libcrun_container_t *container, const char *rootfs,
```

Comment:
are these tabs?

Reply:
Not sure they seems white space only to me, i just did `clang-format` could it be github glitch.

Reply:
Number of white spaces is increased since i changed name from `notify_handler` to `libcrun_container_notify_handler` so clang added more space to match indentation.